common/src/main/java/com/wynntils/core/config/upfixers/impl/GameBarOverlayMoveUpfixer.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.config.upfixers.impl;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.wynntils.core.config.upfixers.ConfigUpfixer;
+import java.util.List;
+
+public class GameBarOverlayMoveUpfixer implements ConfigUpfixer {
+    private static final List<String> KEYS_TO_CHANGE = List.of(
+            "customBarsOverlayFeature.awakenedProgressBarOverlay.flip",
```

Comment:
And it's not like `customBarsOverlayFeature.awakenedProgressBarOverlay.*` could work? Are there configs that should not be converted? The list just looked very repetetive...

Are we thinking we should keep these upfixers forever and ever, or that we should remove them after a while? If it is the latter, I can accept this code, but it it is here to stay forever, I think it needs a bit more shaping up.

Reply:
I would assume we can reset these at release? Or at beta.. I am not sure.

Reply:
Ok.